### PR TITLE
Husky implementation #globalconfig

### PR DESCRIPTION
**Pull request by siggerzz**
Add Husky to run `lint` / `test` scripts before commit

### CODE REVIEW

**Review by charliearlie — DISMISSED** [review]

File: packages/components/organisms/f-registration/babel.config.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ module.exports = api => {
         // use current node version for transpiling test files
         presets.push(['@babel/env', { targets: { node: 'current' } }]);
     }
-
+    // test
```
Comment (1itvinka) [inline packages/components/organisms/f-registration/babel.config.js]: is it something you forgot to remove?

**Review by 1itvinka — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -84,9 +84,18 @@ To run storybook first you must run the following at the the top level
 
 Then run `yarn storybook:serve` to start storybook, which will open on `localhost:6006`.
 
+## Committing Code
+
+We use [Husky](https://github.com/typicode/husky) to manage our git hooks.
+
+[When you make a commit, the following scripts will run as part of our pre-commit hook.] (https://github.com/justeat/fozzie-components/blob/master/.husky/pre-commit)
```
Comment (dandel10n) [inline README.md]: There is a space between `[]` and `()` so it is not valid markdown

File: packages/components/organisms/f-registration/babel.config.js (hunk 1 above)
Comment (siggerzz) [reply packages/components/organisms/f-registration/babel.config.js]: Fixed!

File: README.md (hunk 2 above)
Comment (siggerzz) [reply README.md]: Fixed!

**Review by 1itvinka — APPROVED** [review]

File: .husky/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+_
```
Comment (dandel10n) [inline .husky/.gitignore:1]: Do we need this `_` for gitignore?

Comment (siggerzz) [reply .husky/.gitignore:1]: This was autogenerated by husky when I installed using `yarn`. This folder contains the husky script that's installed based on your OS. `.sh` for MacOS / `.bat` for Windows.

It looks like it's intended 👍🏼

**Review by dandel10n — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -84,9 +84,18 @@ To run storybook first you must run the following at the the top level
 
 Then run `yarn storybook:serve` to start storybook, which will open on `localhost:6006`.
 
+## Committing Code
```
Comment (ashleynolan) [inline README.md:87]: Could you add some specific info on Husky to the [Utility tools](https://justeat.github.io/fozzie-components/@justeat/storybook/index.html?path=/story/documentation-guides-utility-tools--page) docs and then refer to this there then

**Pull request merged**